Feature/mvp of mvp (#1)
vtex-apps/popover-layout: docs/README.md

> 📢 Don't fork this project. Use, [contribute](https://github.com/vtex-apps/awesome-io#contributing), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Don't fork this project. Use, [contribute](https://github.com/vtex-apps/popover-layout), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # DO NOT USE THIS Popover Layout

Reviewer: Maybe it would be better to have a work-in-progress disclaimer above the title instead of this DO NOT USE THIS

This repository provides blocks that can help you create a Dropdown, Select or a Tooltip component.

## Configuration

> 1. Import the popoer layout app to your theme's dependencies in the `manifest.json`, for example:

Reviewer 2: ```suggestion
1. Import the popover layout app to your theme's dependencies in the `manifest.json`, for example:
```

```jsonc
{
  "dependencies": {
    "vtex.popover-layout": "0.x"
  }
}
```

> 2. Now you can use the two blocks exported by vtex.popover-layout. Notice that you need to configure your own popover-layout and pass it as a child of popover-trigger.

Reviewer 2: ```suggestion
2. Now you can use the two blocks exported by `vtex.popover-layout`. Notice that you need to configure your own popover-layout and pass it as a child of popover-trigger.
```

```jsonc

> {

Reviewer 2: I'd re-order these blocks to have them defined before used.

Reviewer 3: I agree with @kaisermann

  "rich-text#link": {
    "props": {
      "text": "\n**Reach us at**\nwww.vtex.com.br",
      "blockClass": "link"
    }
  },
  "popover-layout": {
    "props": {
      "placement": "left"
    },
    "children": [
      "rich-text#link"
    ]
  },
  "rich-text#question": {
    "props": {
      "text": "**Click to open the popoer layout**",
      "blockClass": "question"
    }
  },
  "popover-trigger": {
    "children": [
      "rich-text#question",
      "popover-layout"    
    ]
  }
}
```

### Popover

| Prop name | Type | Description | Default value |
| --- | --- | --- | --- |
| placement | `Placement` | Describes the preferred placement of the popover. If there is no space to put the popover in the placement that you choose it will fit in a fallback position | `'bottom`' |

### PopoverTrigger

| Prop name | Type | Description | Default value |
| --- | --- | --- | --- |

> | trigger | `TriggerMode` | What kind of user action must happen to open the popover | `'click'` |

Reviewer 3: Missing TriggerMode docs
